UniAtHome/UniAtHome.DAL/Repositories/Repository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+using UniAtHome.DAL.Entities;
+using UniAtHome.DAL.Interfaces;
+
+namespace UniAtHome.DAL.Repositories
+{
+    public class Repository<T> : IRepository<T> where T: BaseEntity
```

Comment:
Maybe, we shouldn't use generic repository. As for me, it is not convenient way. What do you think?

Reply:
many functions are common for all the repos (like getbyid, remove, add, etc), so it`s neccessary